### PR TITLE
[W5.7][W16-3]Kenneth Goh Jun Teck

### PR DESCRIPTION
**Pull request by guestuser0123**

### CODE REVIEW

**Comment by nelvintan** [comment]
A fresh enhancement. You might want to consider JUNIT testing.

File: backup.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Jacob Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/owesMoney t/friends
+John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/owesMoney t/friends
+Mary p/5432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/owesMoney t/friends
```
Comment (joanneong) [inline backup.txt:3]: This is fine, but for more realistic testing we can also change the phone numbers, email addresses and addresses for different contacts :P

File: src/seedu/addressbook/commands/BackupCommand.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.commands;
+import seedu.addressbook.storage.StorageFile;
+import seedu.addressbook.storage.StorageFile.InvalidStorageFilePathException;
```
Comment (joanneong) [inline src/seedu/addressbook/commands/BackupCommand.java:3]: Are both of these import statements necessary?

File: src/seedu/addressbook/commands/ImportCommand.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
+package seedu.addressbook.commands;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.storage.StorageFile;
+import seedu.addressbook.storage.StorageFile.InvalidStorageFilePathException;
```
Comment (joanneong) [inline src/seedu/addressbook/commands/ImportCommand.java:5]: Nice ordering maintained for your import statements :)

**Review by joanneong — APPROVED** [review]
Good effort in general :)

However, you can also consider adding JUnit tests for more comprehensive testing (since you only updated the IO tests for now). 

Please close this PR after reading my comments!

**Pull request closed without merging**